README.md

```diff
@@ -96,6 +96,22 @@ If you are new to Rails, follow the [RailsBridge Installfest instructions](http:
       Client Secret from your Github application
     * Don't forget to restart your Rails server so it can see your shiny new GitHub key & secret
 
+#### Docker setup
+
+1. Install docker and docker compose
```

Comment:
If you have time, in a later PR, could you provide a link with instructions on how to do this?